mrp_mass_production_order/tests/test_mrp_mass_production_order.py

```diff
@@ -2,15 +2,27 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 from odoo import Command
-from odoo.tests.common import Form, TransactionCase
+from odoo.tests.common import Form
 
+from odoo.addons.base.tests.common import BaseCommon
 
-class TestMRPMassProductionOrder(TransactionCase):
+
+class TestMRPMassProductionOrder(BaseCommon):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.product_delivery_01 = cls.env.ref("product.product_delivery_01")
-        cls.product_delivery_02 = cls.env.ref("product.product_delivery_02")
+        cls.product_delivery_01 = cls.env["product.product"].create(
+            {
+                "name": "Test product 1",
+            }
+        )
+        cls.product_delivery_02 = cls.env["product.product"].create(
+            {
+                "name": "Test product 2",
+            }
+        )
```

Comment:
Avoid final comma for better line distribution:

```suggestion
        cls.product_delivery_01 = cls.env["product.product"].create(
            {"name": "Test product 1"}
        )
        cls.product_delivery_02 = cls.env["product.product"].create(
            {"name": "Test product 2"}
        )
```